include/seqan3/alphabet/gap/gap.hpp

```diff
@@ -31,117 +31,123 @@
 // DAMAGE.
 //
 // ==========================================================================
-// Author: Hannes Hauswedell <hannes.hauswedell@fu-berlin.de>
-// Author: Marcel Ehrhardt <marcel.ehrhardt@fu-berlin.de>
-// Author: David Heller <david.heller@fu-berlin.de>
-// ==========================================================================
+
+/*!\file
```

Comment:
I don't quite understand the question. If you use the configuration as provided in the wiki, it will produce the same result as using `\file seqan3/alphabet/union_alphabet.hpp` or `\file alphabet/union_alphabet.hpp` or `\file union_alphabet.hpp`.

![doxygen_file_empty_output](https://cloud.githubusercontent.com/assets/103778/26092569/eaced8c0-3a11-11e7-80c9-c6da07a1d7b5.png)